server/src/main/java/org/opensearch/search/aggregations/bucket/BucketsAggregator.java

```diff
@@ -260,23 +259,6 @@ public int size() {
         return result;
     }
 
-    /**
```

Comment:
is it intentional?

Reply:
yes this isn't used anywhere, dead code I found while going through this code